### PR TITLE
Update version number and build info

### PR DESCRIPTION
**Pull request by GarrickHe**
* Bump OS_VERSION to 3.0.0

Signed-off-by: Garrick He <garrick_he@dell.com>

### CODE REVIEW

File: DEBIAN/control [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: opx-core
-Version: 2.3.1
+Version: 2.3.2
```
Comment (jeffyin-dell) [inline DEBIAN/control]: Shouldn't you bump this to 3.0.0? Or do you not want to do that due to future code deliveries to 3.0.0 (like -dev2, -rc0, release)?

Comment (jeffyin-dell) [reply DEBIAN/control]: Even so, this might be confusing if we ever decide to push out a '2.3.2' installer.

**Review by jeffyin-dell — COMMENTED** [review]
Just have one question about the value chosen in Debian/control

File: DEBIAN/control (hunk 1 above)
Comment (heucke) [reply DEBIAN/control]: We could use `3.0.0~b1` or something similar. In Debian, the tilde means the version is less than 3.0.0, but greater than 2.anything.

When ready, bump to `3.0.0`.

Comment (jeffyin-dell) [reply DEBIAN/control]: Sounds good to me

**Review by jeffyin-dell — CHANGES_REQUESTED** [review]
In DEBIAN/control use `3.0.0~b1` rather than `2.3.2`

**Comment by heucke** [comment]
I also need to update the CI for this repo. It doesn't build a stretch package yet. I'll do it this afternoon.

**Comment by heucke** [comment]
I've updated the pipeline. It's a new pipeline, so you'll need to amend and run git-pull-request again to change the commit sha and trigger a build.

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now a95e31fe109a58f72357e9fb6f94e859fe1e67f5

File: DEBIAN/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+opx-core (2.3.2) unstable; urgency=medium
```
Comment (heucke) [inline DEBIAN/changelog]: Need to update the version here too

File: DEBIAN/control [hunk 3]
```diff
@@ -1,5 +1,5 @@
 Package: opx-core
-Version: 2.3.1
+Version: 3.0.0~b1
```
Comment (heucke) [inline DEBIAN/control]: If `OS_VERSION` is `3.0.0-dev1`, let's make the package version `3.0.0~dev1`.

**Review by heucke — CHANGES_REQUESTED** [review]

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now 6d37cd82c98eb0e706aeb8da3b1a910ba30b1559

File: DEBIAN/changelog [hunk 4]
```diff
@@ -1,4 +1,4 @@
-opx-core (2.3.2) unstable; urgency=medium
+opx-core (3.0.0) unstable; urgency=medium
```
Comment (heucke) [inline DEBIAN/changelog]: This version is different from the version in the control file.

**Review by heucke — CHANGES_REQUESTED** [review]

**Comment by GarrickHe** [comment]
Pull-request updated, HEAD is now a8102b4bec3cd7ded2dfcd7d98843348649537a0

**Review by jeffyin-dell — APPROVED** [review]

**Comment by jeffyin-dell** [comment]
@theucke go ahead and merge when you approve these changes.

**Review by heucke — APPROVED** [review]

**Pull request merged**